### PR TITLE
Update and rename How to enable DMARC Reporting for Microsoft Online …

### PR DESCRIPTION
**Pull request by MSFTBen**
…Email Routing Address (MOERA) and parked Domains to How to enable DMARC Reporting for Microsoft Online Email Routing Address (MOERA) and parked Domains.md

### CODE REVIEW

File: microsoft-365/security/office-365-security/step-by-step-guides/How to enable DMARC Reporting for Microsoft Online Email Routing Address (MOERA) and parked Domains.md [hunk 1]
```diff
@@ -27,9 +27,13 @@ This guide is designed to help you configure DMARC for the domains not covered i
 
 ## What you’ll need
 
-- Microsoft 365 admin centre and access to your DNS provider hosting your domains
+- Microsoft 365 admin center and access to your DNS provider hosting your domains
 - Sufficient permissions as Global Admin to make the appropriate changes in the Microsoft 365 Admin Center
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/step-by-step-guides/How to enable DMARC Reporting for Microsoft Online Email Routing Address (MOERA) and parked Domains.md:31]:
```suggestion
- Sufficient permissions as global admin to make the appropriate changes in Microsoft 365 admin center
```

File: microsoft-365/security/office-365-security/step-by-step-guides/How to enable DMARC Reporting for Microsoft Online Email Routing Address (MOERA) and parked Domains.md [hunk 2]
```diff
@@ -27,9 +27,13 @@ This guide is designed to help you configure DMARC for the domains not covered i
...
 - 10 Minutes to complete the following steps
+- DMARC report provider 
+
+> [!TIP]
+> Visit the [MISA catalog](https://www.microsoft.com/misapartnercatalog) to view more third-party vendors offering DMARC reporting for Microsoft 365. See [IETF.org's 'Domain-based Message Authentication, Reporting, and Conformance (DMARC)'](https://datatracker.ietf.org/doc/html/rfc7489) for more information on DMARC 'rua' addresses.
 
 ## Activate DMARC for MOERA Domain
 1.  Login to the [Microsoft 365 Admin Center](https://admin.microsoft.com).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/step-by-step-guides/How to enable DMARC Reporting for Microsoft Online Email Routing Address (MOERA) and parked Domains.md:39]:
```suggestion
1. Sign in to [Microsoft 365 admin center](https://admin.microsoft.com).
```

File: microsoft-365/security/office-365-security/step-by-step-guides/How to enable DMARC Reporting for Microsoft Online Email Routing Address (MOERA) and parked Domains.md [hunk 3]
```diff
@@ -44,13 +48,13 @@ This guide is designed to help you configure DMARC for the domains not covered i
 1.  Press **Save**.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/step-by-step-guides/How to enable DMARC Reporting for Microsoft Online Email Routing Address (MOERA) and parked Domains.md:48]:
```suggestion
1.  Select **Save**.
```

File: microsoft-365/security/office-365-security/step-by-step-guides/How to enable DMARC Reporting for Microsoft Online Email Routing Address (MOERA) and parked Domains.md [hunk 4]
```diff
@@ -44,13 +48,13 @@ This guide is designed to help you configure DMARC for the domains not covered i
...
 
 ## Active DMARC for parked domains
-1.  Check if SPF is already configured for your parked domain, following this guide: [Set up SPF to help prevent spoofing - Office 365 | Microsoft Docs](https://docs.microsoft.com/microsoft-365/security/office-365-security/set-up-spf-in-office-365-to-help-prevent-spoofing?view=o365-worldwide#how-to-handle-subdomains)
+1.  Check if SPF is already configured for your parked domain, following this guide: [Set up SPF to help prevent spoofing - Office 365 | Microsoft Docs](https://docs.microsoft.com/microsoft-365/security/office-365-security/set-up-spf-in-office-365-to-help-prevent-spoofing#how-to-handle-subdomains)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/step-by-step-guides/How to enable DMARC Reporting for Microsoft Online Email Routing Address (MOERA) and parked Domains.md:51]:
```suggestion
1.  Check if SPF is already configured for your parked domain, following this guide: [Set up SPF to help prevent spoofing - Office 365 | Microsoft Docs](/microsoft-365/security/office-365-security/set-up-spf-in-office-365-to-help-prevent-spoofing#how-to-handle-subdomains).
```

File: microsoft-365/security/office-365-security/step-by-step-guides/How to enable DMARC Reporting for Microsoft Online Email Routing Address (MOERA) and parked Domains.md [hunk 5]
```diff
@@ -44,13 +48,13 @@ This guide is designed to help you configure DMARC for the domains not covered i
...
 1.  Contact your DNS Domain provider.
-1.  Ask to add this DMARC txt record with your appropriate email addresses "v=DMARC1; p=reject; rua=mailto:d@rua.contoso.com;ruf=mailto:d@ruf.contoso.com"
+1.  Ask to add this DMARC txt record with your appropriate email reporting addresses "v=DMARC1; p=reject; rua=mailto:d@rua.contoso.com;ruf=mailto:d@ruf.contoso.com"
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/step-by-step-guides/How to enable DMARC Reporting for Microsoft Online Email Routing Address (MOERA) and parked Domains.md:53]:
```suggestion
1.  Ask to add this DMARC txt record with your appropriate email reporting addresses "v=DMARC1; p=reject; rua=mailto:d@rua.contoso.com;ruf=mailto:d@ruf.contoso.com".
```

File: microsoft-365/security/office-365-security/step-by-step-guides/How to enable DMARC Reporting for Microsoft Online Email Routing Address (MOERA) and parked Domains.md [hunk 6]
```diff
@@ -44,13 +48,13 @@ This guide is designed to help you configure DMARC for the domains not covered i
...
 
 ## Next Steps
 Wait until the DNS changes are propagated and try to spoof the configured domains. Check if the attempt is blocked based in the DMARC record, and you receive a DMARC report.
 
 ## More Information
-[Set up SPF to help prevent spoofing - Office 365 | Microsoft Docs](https://docs.microsoft.com/microsoft-365/security/office-365-security/set-up-spf-in-office-365-to-help-prevent-spoofing?view=o365-worldwide)
-[Use DMARC to validate email, setup steps - Office 365 | Microsoft Docs](https://docs.microsoft.com/microsoft-365/security/office-365-security/use-dmarc-to-validate-email?view=o365-worldwide)
+[Set up SPF to help prevent spoofing - Office 365 | Microsoft Docs](https://docs.microsoft.com/microsoft-365/security/office-365-security/set-up-spf-in-office-365-to-help-prevent-spoofing)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/step-by-step-guides/How to enable DMARC Reporting for Microsoft Online Email Routing Address (MOERA) and parked Domains.md:59]:
```suggestion
[Set up SPF to help prevent spoofing - Office 365 | Microsoft Docs](/microsoft-365/security/office-365-security/set-up-spf-in-office-365-to-help-prevent-spoofing)
```

File: microsoft-365/security/office-365-security/step-by-step-guides/How to enable DMARC Reporting for Microsoft Online Email Routing Address (MOERA) and parked Domains.md [hunk 7]
```diff
@@ -44,13 +48,13 @@ This guide is designed to help you configure DMARC for the domains not covered i
...
+[Use DMARC to validate email, setup steps - Office 365 | Microsoft Docs](https://docs.microsoft.com/microsoft-365/security/office-365-security/use-dmarc-to-validate-email)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/step-by-step-guides/How to enable DMARC Reporting for Microsoft Online Email Routing Address (MOERA) and parked Domains.md:60]:
```suggestion
[Use DMARC to validate email, setup steps - Office 365 | Microsoft Docs](/microsoft-365/security/office-365-security/use-dmarc-to-validate-email)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @MSFTBen, kindly also commit the few additional suggested changes (even though they are mostly not related to your changes). Many thanks.

**Comment by yogkumgit** [comment]
@MSFTBen Gentle reminder

**Review by MSFTTracyP — APPROVED** [review]
Thanks

**Pull request merged**